packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_payback_loan.test.ts

```diff
@@ -144,13 +144,13 @@ async function setup (): Promise<void> {
 
   // create vault for taking large loan tokens
   const aliceVaultAddr = await alice.generateAddress()
-  const aliceVaultId = await alice.rpc.loan.createVault({
+  const aliceVaultId = await alice.rpc.vault.createVault({
     ownerAddress: aliceVaultAddr,
     loanSchemeId: 'default'
   })
   await alice.generate(1)
 
-  await alice.rpc.loan.depositToVault({
+  await alice.rpc.vault.depositToVault({
```

Comment:
This specific tests should not need to retain its loan alternative as it is meant to test `paybackLoan`.

Reply:
As per https://github.com/DeFiCh/jellyfish/pull/1251#discussion_r833853364, will now revert this back to using loan until full deprecation